applications/RANSApplication/python_scripts/formulations/fractional_step/fractional_step_velocity_pressure_rans_formulation.py

```diff
@@ -0,0 +1,344 @@
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+import KratosMultiphysics.RANSApplication as KratosRANS
+
+# import formulation interface
+from KratosMultiphysics.RANSApplication.formulations.rans_formulation import RansFormulation
+
+# import utilities
+from KratosMultiphysics.RANSApplication import RansVariableUtilities
+from KratosMultiphysics.RANSApplication import RansCalculationUtilities
+from KratosMultiphysics.RANSApplication.formulations.utilities import CreateRansFormulationModelPart
+from KratosMultiphysics.RANSApplication.formulations.utilities import CalculateNormalsOnConditions
+from KratosMultiphysics.RANSApplication.formulations.utilities import GetConvergenceInfo
+from KratosMultiphysics.RANSApplication.formulations.utilities import GetKratosObjectType
+from KratosMultiphysics.RANSApplication.formulations.utilities import InitializePeriodicConditions
+
+
+class FractionalStepVelocityPressureRansFormulation(RansFormulation):
+    def __init__(self, model_part, settings):
+        """Incompressible Fractional Step Navier Stokes formulation
+
+        This RansFormulation solves VELOCITY, and PRESSURE with Fractional Step formulated
+        incompressible Navier-Stokes equation.
+
+        This supports both steady and transient problems, transient with time integration scheme.
+        It uses wall functions at walls, therefore SLIP flag must be True on all walls (in both conditions and nodes)
+
+        Args:
+            model_part (Kratos.ModelPart): ModelPart to be used in the formulation.
+            settings (Kratos.Parameters): Settings to be used in the formulation.
+        """
+        super().__init__(model_part, settings)
+
+        ##settings string in json format
+        default_settings = Kratos.Parameters("""
+        {
+            "formulation_name": "fractional_step",
+            "predictor_corrector": false,
+            "maximum_velocity_iterations": 3,
+            "maximum_pressure_iterations": 3,
+            "velocity_tolerance": 1e-3,
+            "pressure_tolerance": 1e-2,
+            "dynamic_tau": 0.01,
+            "oss_switch": 0,
+            "echo_level": 0,
+            "time_order": 2,
+            "compute_reactions": false,
+            "reform_dofs_at_each_step": false,
+            "pressure_linear_solver_settings":  {
+                "solver_type"                    : "amgcl",
+                "max_iteration"                  : 200,
+                "tolerance"                      : 1e-6,
+                "provide_coordinates"            : false,
+                "smoother_type"                  : "ilu0",
+                "krylov_type"                    : "cg",
+                "gmres_krylov_space_dimension"   : 100,
+                "use_block_matrices_if_possible" : false,
+                "coarsening_type"                : "aggregation",
+                "scaling"                        : true,
+                "verbosity"                      : 0
+            },
```

Comment:
Shouldn't the default pressure linear solver be the CG one?

Reply:
I was copying the same from FluidApp, apparently these are being used there.

Reply:
OK. Probably we changed it at some point.